projects/1.18-fabric/assets/fabric-seasons/seasons/lang/zh_cn.json

```
@@ -0,0 +1,17 @@
+{
+  "item.seasons.season_calendar": "季节历",
+  "block.seasons.season_detector": "季节探测器",
+  "block.seasons.greenhouse_glass": "温室玻璃",
+  "tooltip.seasons.spring": "§a春",
+  "tooltip.seasons.summer": "§6夏",
+  "tooltip.seasons.fall": "§4秋",
+  "tooltip.seasons.winter": "§b冬",
+  "tooltip.seasons.calendar_info_1": "§9当前季节：",
+  "tooltip.seasons.calendar_info_2": " §7剩余天数到",
```

```suggestion
  "tooltip.seasons.calendar_info_2": "§7天后进入",
```
作者不给改语序的选项，~~打一下~~
也要考虑一下像春夏这种需不需要改成春季夏季

---

嘶，我去瞄一眼其他类似mod怎么翻译的